demo/lib/distro.bash

```diff
@@ -466,6 +466,9 @@ fedora-install-k8s() {
     local yumkey="https://packages.cloud.google.com/yum/doc/yum-key.gpg"
     local rpmkey="https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg"
 
+    vm-command "( lsmod | grep -q br_netfilter ) || { echo br_netfilter > /etc/modules-load.d/50-br_netfilter.conf; modprobe br_netfilter; }"
+    vm-command "echo 1 > /proc/sys/net/ipv4/ip_forward"
```

Comment:
I just noticed that we need to save the setting to a sysctl config file that gets loaded during boot in order this to work as expected. I will send a fixed version of this one.

Reply:
I think this probably could/should be done with(in) fedora-(35-)specific `bootstrap-commands()`...

Reply:
Indeed, the default-bootstrap-commands() even sets the ip_forward already. I am just not seeing it set in my test setup, need to investigate why the default func is not run.